elasticsearch/es_client_v7.go

```diff
@@ -301,3 +302,85 @@ func (es *ESClientV7) GetTotalDiskUsage(ctx context.Context) (string, error) {
 
 	return totalDiskUsage, nil
 }
+
+func (es *ESClientV7) getDBUserRole(ctx context.Context) (error, bool) {
+	req := esapi.SecurityGetRoleRequest{
+		Name: []string{CustomUser},
+	}
+	res, err := req.Do(ctx, es.client.Transport)
+	defer func(Body io.ReadCloser) {
+		err := Body.Close()
+		if err != nil {
+			klog.Errorf("failed to close response body from GetDBUserRole", err)
+		}
+	}(res.Body)
+
+	if err != nil {
+		fmt.Println("faced error while making request in GetDBUserRole function")
```

Comment:
Error message here, should be "Failed to get DB user Role"
Use Klog.Errorf for logging